cdi/common-jakarta/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.infinispan</groupId>
+        <artifactId>infinispan-cdi-parent</artifactId>
+        <version>14.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>infinispan-cdi-common-jakarta</artifactId>
+    <name>Infinispan CDI JakartaEE support common</name>
```

Comment:
Kind of a nitpick but "JakartaEE" as one word is not a thing, use "Jakarta EE" instead. Applies to many places. Simple find & replace should do the trick.

Reply:
Updated, thanks!